Running unit tests fails with: ERROR: Missing environment variable: HOME

<!---
Verify first that your issue/request is not already reported on GitHub.
Also test if the latest release, and master branch are affected too.
-->

##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME
N/A

##### ANSIBLE VERSION
```
ansible 2.3.0 (devel 4673a9c9c2) last updated 2017/01/18 17:51:21 (GMT +100)
  config file = 
  configured module search path = Default w/o overrides
```

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
Arch Linux

##### SUMMARY
Running the unit tests with `tox -epy27` (any py{2,3}*) fails with:

```
test/runner/ansible-test units -v --python 2.7
ERROR: Missing environment variable: HOME
make: *** [Makefile:118: tests] Error 1
ERROR: InvocationError: '/usr/bin/make tests'
```

Full error: http://pastebin.com/Qs2xbdBj

##### STEPS TO REPRODUCE
Just clone the ansible repository and run ``tox -epy27``

##### EXPECTED RESULTS
The unit tests to be executed

##### ACTUAL RESULTS
The execution of the unit tests doesn't start

--- a/tox.ini
+++ b/tox.ini
@@ -16,6 +16,10 @@ commands =
     py27: python -m compileall -fq -x 'test/samples' lib test contrib
     py3{5,6}: python -m compileall -fq -x 'test/samples|lib/ansible/modules' lib test contrib
     make tests
+passenv =
+    # Pass HOME to the test environment to avoid the missing HOME env
+    # variable error. See issue: #20424
+    HOME
 
 
 [flake8]

